app/scripts/background.js

```diff
@@ -69,6 +69,8 @@ function setupController (initState) {
     // platform specific api
     platform,
   })
+
+  controller.on('locked', platform.reload.bind(platform))
```

Comment:
Do we really think this is needed? It's kinda cool that we never crash the app on purpose anymore.